ui/src/components/scoreboard/Scoreboard.tsx

```diff
@@ -25,6 +27,33 @@ const Scoreboard = () => {
     const { isLoading: isStatsLoading, data: statData } = useFetchAllStats();
     const navigate = useNavigate();
 
+    const [selectedOffice, setSelectedOffice] = useState<string>("All Offices");
+
+    const offices = ["All Offices", "Bristol", "Newcastle"];
+
+    const formatOptions = (options: string[]) => {
+      if (options) {
+        const formattedOptions = options.map((elem) => {
+          return {
+            label: elem,
+            value: elem,
+          };
+        });
+        return formattedOptions;
+      }
+    };
+  
+    const filteredUsers = (): IUser[] => {
+      if (!userData) return []; 
+      if (selectedOffice === "All Offices") {
+        return [...userData]
+      }
+      if (selectedOffice === "Newcastle") {
```

Comment:
not a _huge_ fan of this, and will be resolved if we go back through the data to add Newcastle to all current users.
but, would it be better to have a getter for 'userOffice' which defaults to Newcastle if undefined, and remove this branching code?